deploy/kubernetes/base/node_setup/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace:
+  gce-pd-csi-driver
+resources:
+- linux/node.yaml
+- windows/node.yaml
```

Comment:
This means windows deployment automatically gets added to stable. Is that what we want? Or should we first add windows to the alpha overlay, and eventually when it becomes stable, we add it to the base?

Reply:
Another alternative is to have the overlays directly specify the os/node.yaml resources that they want.

Reply:
Thanks to @Liujingfang1, we can make image update lists reusable for different versions. I think with this change, it can customize every possible way easily.